Resources/config/services_dev.yaml

```diff
@@ -20,13 +20,14 @@ services:
     class: Ise\WebSecurityBundle\EventSubscriber\FetchMetadataRequestSubscriber
     arguments: 
       $fetchMetadataPolicy: '@?Ise\WebSecurityBundle\Policies\FetchMetadataDefaultPolicy'
+      $configProvider: '@Ise\WebSecurityBundle\Options\ConfigProvider'
+    tags:
+      - { name: monolog.logger, channel: security }
```

Comment:
This is more of a BC thing than anything else. monolog will take over from Psr\Log\LoggerInterface if it's available within the symfony application, if not it will still function the same.